index.html

```diff
@@ -17,10 +17,35 @@
   </head>
   <body>
     <header>
-      <div class="container"></div>
+      <div class="container" id="header">
+        <h1 class="column-full">Code Journal</h1>
+      </div>
     </header>
     <main class="container">
-      <div data-view="entry-form"></div>
+      <h1 class="column-full" id="new-entry">New Entry</h1>
+      <div data-view="entry-form">
```

Comment:
Line 25 needs to be inside the `div` on line 26. The "New Entry" text is only associated with the `"entry-form" view. Will come into play in the next feature